tests/console/parsec.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test parsec service with parsec-tool
+# Maintainer: Guillaume Gardet <guillaume@opensuse.org>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    # Install requirements
+    select_console 'root-console';
+    my $pkg_list = "parsec parsec-tool";
+    zypper_call("in $pkg_list");
+    systemctl 'start parsec';
+
+    # Add user to 'parsec-clients' group and force relogin
+    assert_script_run("usermod -a -G parsec-clients $testapi::username");
+    select_console('user-console');
+    type_string("exit\n");
+    # Exit from serial_terminal before reseting consoles, as a workaround until https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11405 get merged
```

Comment:
You could also just run `newgrp parsec-clients`

Reply:
`newgrp` is a bit far from the real use case.

Reply:
I always use `newgrp` instead of logging out and in again, so it's definitely closer to how I would use it :P

That it avoids an openQA bug and saves time is a bonus!

Reply:
I am giving it a try.